test/indexer/test_hca_indexer.py

```diff
@@ -1817,33 +1817,56 @@ def test_related_files_field_exclusion(self):
                                      })
         self.assertEqual(0, hits["hits"]["total"])
 
-    def test_samples_downstream_entities(self):
+    def test_downstream_entities(self):
         """
-        Verify that samples include bam files from stitched subgraphs
+        Verify that samples and cell_suspensions include analysis files from
+        stitched subgraphs
         """
-        bundle = self.bundle_fqid(uuid='79fa91b4-f1fc-534b-a935-b57342804a70',
-                                  version='2020-12-10T10:30:00.000000Z')
-        self._index_canned_bundle(bundle)
+        bundle_fqid = self.bundle_fqid(uuid='79fa91b4-f1fc-534b-a935-b57342804a70',
+                                       version='2020-12-10T10:30:00.000000Z')
+        bundle = self._load_canned_bundle(bundle_fqid)
 
-        def is_sample_aggregate(hit):
-            entity_type, aggregate = self._parse_index_name(hit)
-            return entity_type == 'samples' and aggregate
+        expected_cell_count = 123
+        # The bundles that motivated this test case lack the
+        # `estimated_cell_count` field, so we inject it here to avoid nulls in
+        # the index.
+        for document_name, document in bundle.metadata_files.items():
+            if document_name.startswith('cell_suspension'):
+                document['estimated_cell_count'] = expected_cell_count
 
-        samples = [
-            hit['_source']['contents']
-            for hit in self._get_all_hits()
-            if is_sample_aggregate(hit)
-        ]
+        self._index_bundle(bundle)
+
+        def get_aggregates(hits, type):
+            for hit in hits:
+                entity_type, aggregate = self._parse_index_name(hit)
+                if entity_type == type and aggregate:
+                    yield hit['_source']['contents']
+
+        all_hits = self._get_all_hits()
```

Comment:
```suggestion
        hits = self._get_all_hits()
```

If the reason you named it that is that you want to avoid shadowing: I have the shadowing inspection warning disabled in PyCharm. I think nested functions can safely shadow variables in the containing scope.